src/test/java/io/appium/java_client/android/AndroidDriverTest.java

```diff
@@ -43,276 +43,281 @@
 
 public class AndroidDriverTest extends BaseAndroidTest {
 
-    @Test
```

Comment:
Can you not alter the indentation and list only added test?

Reply:
@SrinivasanTarget sure, reverted the file and added my test